app/test/unit/git/apply-test.ts

```diff
@@ -0,0 +1,49 @@
+import { GitProcess } from 'dugite'
+import { setupTwoCommitRepo } from '../../helpers/repositories'
+import { Repository } from '../../../src/models/repository'
+import { checkPatch } from '../../../src/lib/git'
+import {
+  cloneLocalRepository,
+  makeCommit,
+} from '../../helpers/repository-scaffolding'
+
+describe('git/apply', () => {
+  describe('checkPatch', () => {
+    describe('on related repository without conflicts', () => {
+      let repository: Repository
+      let patch: string
+      beforeEach(async () => {
+        const originalRepo = await setupTwoCommitRepo()
+        repository = await cloneLocalRepository(originalRepo)
+        await makeCommit(originalRepo, {
+          entries: [{ path: 'just-okay-file', contents: 'okay' }],
+        })
+        ;({ stdout: patch } = await GitProcess.exec(
```

Comment:
Can we make this variable assignment more readable? What about something like this?

```ts
const result = await GitProcess.exec(
  ['format-patch', '--stdout', 'HEAD~'],
  originalRepo.path
)
patch = result.stdout
```

Reply:
cce48cf93d5ad31f9d8619ad7502381a3366ec1f

Reply:
🤦‍♀

Reply:
dab8b4815f91980e03221a9dda052f0edbbda71b